packages/graph-editor/src/components/contextMenus/selectionContextMenu.tsx

```diff
@@ -27,12 +27,10 @@ export const SelectionContextMenu = ({ id, nodes }: INodeContextMenuProps) => {
   const deleteNode = useAction('deleteNode');
   const duplicateNodes = useAction('duplicateNodes');
   const { copySelectedNodes } = useCopyPaste();
-  const trigger = useToast();
+  const toast = useToast();
 
   //Note that we use a filter here to prevent getting nodes that have a parent node, ie are part of a group
-  const selectedNodes = nodes.filter(
-    (node) => node.selected && !node.parentNode,
-  );
+  const selectedNodes = nodes.filter((node) => node.selected && !node.parentId);
```

Comment:
I'm assuming this was a typo initially or changed spec with the parentNode / parentId issue here?